app/src/main/java/com/hover/stax/ApplicationInstance.java

```diff
@@ -45,6 +51,29 @@ private void setLogger() {
             Timber.uprootAll();
         }
     }
+
+    private void initSentry(){
```

Comment:
Do we have to use manual initialization for Stax?

Reply:
From my tests, it seems the `SDK` configurations override Stax's and the events aren't recorded.

Reply:
Cool, was just curious. It does work this way?